Misc/NEWS.d/next/C API/2024-04-20-14-05-12.gh-issue-118124.K-TPl4.rst

```diff
@@ -0,0 +1 @@
+Deprecates :c:macro:`Py_BUILD_ASSERT`.  Passing non-constant expression to :c:macro:`Py_BUILD_ASSERT` or :c:macro:`Py_BUILD_ASSERT_EXPR` now makes compilation failed.
```

Comment:
Sorry, I though the blurb_it web app could split lines automatically since there's no column number displayed and it's hard to count characters in its non-mono-font editor with eyes.
Do I need to split to multiple lines? If I do, should I force-push to make history clean?

Reply:
Splitted and force-push with other changes.